README.md

```
@@ -57,6 +57,11 @@ winget install FiloSottile.mkcert
 
 詳しくは [apps/server/README.md](./apps/server/README.md) を参照。
 
+### リポジトリのclone
+任意のディレクトリ（普段使ってる開発フォルダ等）に以下のコマンドでclone
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)